src/test/java/de/tum/in/www1/artemis/ArchitectureTest.java

```diff
@@ -216,4 +222,32 @@ public void check(JavaMethod item, ConditionEvents events) {
             }
         };
     }
+
+    private static final Set<String> SQL_KEYWORDS = Set.of("SELECT", "UPDATE", "SET", "DELETE", "DISTINCT", "EXISTS", "FROM", "WHERE", "LEFT", "OUTER", "INNER", "JOIN", "FETCH",
+            "TREAT", "AND", "OR", "AS", "ON", "ORDER", "BY", "ASC", "DSC", "GROUP", "COUNT", "SUM", "AVG", "MAX", "MIN", "IS", "NOT", "FALSE", "TRUE", "NULL", "LIKE", "IN",
+            "BETWEEN", "HAVING", "EMPTY", "MEMBER", "OF", "UPPER", "LOWER", "TRIM");
+
+    private ArchCondition<JavaMethod> useUpperCaseSQLStyle() {
+        return new ArchCondition<>("@Query content should follow the style guide") {
```

Comment:
```suggestion
        return new ArchCondition<>("follow the style guide") {
```
otherwise the constructed string of the ArchUnit response will be "Methods that are annotated with a Query Annotation should @Query content should follow the style guide")

Though, I would actually suggest phrasing it directly "should use uppercase SQL style" or something like that

Reply:
> Though, I would actually suggest phrasing it directly "should use uppercase SQL style" or something like that.

While the test only looks for capitalization, I explicitly wrote "follow the style guide" to remind developers to check for also the other style rules that are not tested here.

Reply:
Oh, but in that case, you should add a `.because(...)` clause to the actual query construction above. This only specifies the meaning of the condition.
Then here `use upper case SQL style` and above `.because("@Query content should follow the style guide")`